source/_integrations/tasmota.markdown

```diff
@@ -24,14 +26,16 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
 
 ## Supported Features
 
-Lights, relays, sensors, switches and buttons are supported.
+Tasmota Buttons, Fans, Lights, relays, Sensors, Shutters and Switches are supported.
 
-- Lights will be added as Home Assistant `light` entities. Single channel Dimmers, RGB lights, RGB lights with Color Temperature control and RGB lights with White control are supported.
-- Relays will be added as Home Assistant `switch` entities, if `SetOption30 = 0`. If `SetOption30 = 1`, relays will be added as `light` entities.
-- Sensors will be added as Home Assistant `sensor` entities.
-- Switches will be added as Home Assistant `binary_sensor` entities or `automation triggers` depending by the `switchmode` used when `SetOption114` is enabled.
-- Buttons will be added as Home Assistant `automation triggers` when `SetOption73` is enabled.
+- Tasmota Buttons will be added as Home Assistant `automation triggers` when `SetOption73` is enabled.
+- Tasmota Lights will be added as Home Assistant `light` entities. Single channel Dimmers, RGB lights, RGB lights with Color Temperature control and RGB lights with White control are supported.
+- Tasmota Relays will be added as Home Assistant `switch` entities, if `SetOption30 = 0`. If `SetOption30 = 1`, relays will be added as `light` entities.
+- Tasmota Sensors will be added as Home Assistant `sensor` entities.
+- Tasmota Shutters will be added as Home Assistant `cover` entities.
```

Comment:
Please note `Shutter mode 5` (configuration with a single relay and a PWM) is not yet supported by HaTasmota. My bad, I need to finish the shutter mode for Tuya MCU to see what is the best way to implement it on the integration topic.

Reply:
OK, how to express this limitation? Should we say shutters which are not `mode 5` and not Tuya are supported?

Reply:
Shutter mode 5 is rarely used but for TuyaMCU will be perfect. I have no idea when I will be able to add it on Tasmota, even so will be not on 9.2. Perhaps if there will be a 9.2.1?
How about adding:
```
Currently only Shutter modes 1 to 4 are supported.
```

Reply:
Thanks, updated.